.gitignore

```diff
@@ -51,3 +51,5 @@ android/app/libs
 fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
+
+src/js/keys.js
```

Comment:
maybe this should be in the root, where you would normally expect env files to be?